Makefile

```diff
@@ -27,14 +27,14 @@ build: golang
 	@mkdir -p bin
 	go build -ldflags "${LFLAGS}" -o bin/${NAME}
 
-release: clean golang deps
```

Comment:
I assume some other PR maintains the Dockerfile at https://github.com/jboss-dockerfiles/keycloak/tree/master/gatekeeper (same for k8s charts)

Reply:
@fredbi not sure if I follow you on this. Could you please elaborate?

Reply:
oh you're right. I've seen that the Dockerfile there is building from binary release, not source.
My bad. It is constrasting with our own Dockerfiles which build most things from source...

Reply:
Absolutely no worries, thanks for reviewing it.